<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QMainWindow" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1430</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="progressBoard">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1101</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="colorBoard">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>1101</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>20</y>
      <width>291</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>TintsandTells</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="menu">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>170</y>
      <width>241</width>
      <height>591</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="startGame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Game!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ability_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ability_1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ability_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ability_2</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="timer">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>450</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="timerLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>390</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Timer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ability_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ability_3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="forfeit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>530</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Forfeit</string>
     </property>
    </widget>
    <widget class="QLabel" name="abilityLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Abilities</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1430</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
